pkg/utils/db_test.go

```diff
@@ -17,11 +17,198 @@ import (
+		"mysql-bin.000009", 11232, nil, nil,
+	)
+	mock.ExpectQuery(`SHOW MASTER STATUS`).WillReturnRows(rows)
+	rows = mock.NewRows([]string{"Variable_name", "Value"}).AddRow("gtid_binlog_pos", "1-2-100")
+	mock.ExpectQuery(`SHOW GLOBAL VARIABLES LIKE 'gtid_binlog_pos'`).WillReturnRows(rows)
+
+	pos, gs, err = GetMasterStatus(db, "mariadb")
+	c.Assert(err, IsNil)
+	c.Assert(pos, Equals, gmysql.Position{
+		Name: "mysql-bin.000009",
+		Pos:  11232,
+	})
+	c.Assert(gs.String(), Equals, "1-2-100")
+	c.Assert(mock.ExpectationsWereMet(), IsNil)
+}
+
+func (t *testDBSuite) TestGetMariaDBGtidDomainID(c *C) {
+	db, mock, err := sqlmock.New()
+	c.Assert(err, IsNil)
+
+	rows := mock.NewRows([]string{"Variable_name", "Value"}).AddRow("gtid_domain_id", 101)
+	mock.ExpectQuery(`SHOW GLOBAL VARIABLES LIKE 'gtid_domain_id'`).WillReturnRows(rows)
+
+	dID, err := GetMariaDBGtidDomainID(db)
+	c.Assert(err, IsNil)
+	c.Assert(dID, Equals, uint32(101))
+	c.Assert(mock.ExpectationsWereMet(), IsNil)
+}
+
+func (t *testDBSuite) TestGetServerUUID(c *C) {
+	db, mock, err := sqlmock.New()
+	c.Assert(err, IsNil)
+
+	// MySQL
+	rows := mock.NewRows([]string{"Variable_name", "Value"}).AddRow("server_uuid", "074be7f4-f0f1-11ea-95bd-0242ac120002")
+	mock.ExpectQuery(`SHOW GLOBAL VARIABLES LIKE 'server_uuid'`).WillReturnRows(rows)
+	uuid, err := GetServerUUID(db, "mysql")
+	c.Assert(err, IsNil)
+	c.Assert(uuid, Equals, "074be7f4-f0f1-11ea-95bd-0242ac120002")
+	c.Assert(mock.ExpectationsWereMet(), IsNil)
+
+	// MariaDB
+	rows = mock.NewRows([]string{"Variable_name", "Value"}).AddRow("gtid_domain_id", 123)
+	mock.ExpectQuery(`SHOW GLOBAL VARIABLES LIKE 'gtid_domain_id'`).WillReturnRows(rows)
+	rows = mock.NewRows([]string{"Variable_name", "Value"}).AddRow("server_id", 456)
+	mock.ExpectQuery(`SHOW GLOBAL VARIABLES LIKE 'server_id'`).WillReturnRows(rows)
+	uuid, err = GetServerUUID(db, "mariadb")
+	c.Assert(err, IsNil)
+	c.Assert(uuid, Equals, "123-456")
+	c.Assert(mock.ExpectationsWereMet(), IsNil)
+}
+
+func (t *testDBSuite) TestGetParser(c *C) {
+	var (
+		DDL1 = `ALTER TABLE tbl ADD COLUMN c1 INT`
+		DDL2 = `ALTER TABLE tbl ADD COLUMN 'c1' INT`
+	)
+
+	db, mock, err := sqlmock.New()
+	c.Assert(err, IsNil)
+
+	// no `ANSI_QUOTES`
+	rows := mock.NewRows([]string{"Variable_name", "Value"}).AddRow("sql_mode", "")
+	mock.ExpectQuery(`SHOW VARIABLES LIKE 'sql_mode'`).WillReturnRows(rows)
+	p, err := GetParser(db)
+	c.Assert(err, IsNil)
+	_, err = p.ParseOneStmt(DDL1, "", "")
+	c.Assert(err, IsNil)
+	_, err = p.ParseOneStmt(DDL2, "", "")
+	c.Assert(err, NotNil)
+	c.Assert(mock.ExpectationsWereMet(), IsNil)
+
+	// `ANSI_QUOTES`
+	rows = mock.NewRows([]string{"Variable_name", "Value"}).AddRow("sql_mode", "ANSI_QUOTES")
+	mock.ExpectQuery(`SHOW VARIABLES LIKE 'sql_mode'`).WillReturnRows(rows)
+	p, err = GetParser(db)
+	c.Assert(err, IsNil)
+	_, err = p.ParseOneStmt(DDL1, "", "")
+	c.Assert(err, IsNil)
+	_, err = p.ParseOneStmt(DDL2, "", "")
```

Comment:
is this test intended to have a SQL like `ALTER TABLE tbl ADD COLUMN "c1" INT` and test it could parse?